### PR TITLE
feat: implement activities home page

### LINKED ISSUE #26: 🚀 Feature: Activities Home Page
Make the activites home screen pretty

* [x] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by BryanPan342**
* implement designs for the activities home screen
* descriptive urls

**Note**:
* I did not add the bottom text because unfortunately it usually doesn't fit into the page comfortably
* I also did not add the recommended stuff because it didn't seem necessary and truthfully I couldn't find a clean way to implement it, BUT if this is something that we want, I'll try to figure something out

Fixes: #26

### CODE REVIEW

File: src/components/Activities.tsx [hunk 1]
```diff
@@ -9,52 +9,64 @@ import {
...
 import { HeaderSections } from './shared/PlaynetConstants';
 
-function Activities(): JSX.Element {
-  const { url, path } = useRouteMatch();
+import './styles/Activities.scss';
+
+/**
+ * The following is a mapping between an activity and its descirption
+ */
+const ACTIVITES: {[key: string]: string} = {
```
Comment (reginawang99) [inline src/components/Activities.tsx:21]: nit: ACTIVITIES

File: src/components/Activities.tsx [hunk 2]
```diff
@@ -9,52 +9,64 @@ import {
...
-          <Route path="/">
-            <ActivityHome url={url} />
-          </Route>
-        </Switch>
-      </Base>
+    <div id={'activity-home'}>
+      <h2>Activities</h2>
+      <nav id={'activity-card-container'}>
+        {Object.keys(ACTIVITES).map((path, i) =>
+          <Link to={`${url}/${path}`} key={`activites-card-${i}`}>
```
Comment (reginawang99) [inline src/components/Activities.tsx:38]: nit
```suggestion
          <Link to={`${url}/${path}`} key={`activities-card-${i}`}>
```

Comment (reginawang99) [reply src/components/Activities.tsx:38]: also, thats cool btw :0

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
i think there might be a bug somewhere in the webpack code 

https://deploy-preview-25--youthful-engelbart-1ad9d2.netlify.app/activities/mind-reading doesnt work. (idk if this helps but i saw this: https://github.com/vuejs-templates/pwa/issues/165) and also, in the other pr i had up, i had to add in `    publicPath: '/',` to the prod webpack

File: src/components/Activities.tsx (hunk 1 above)
Comment (BryanPan342) [reply src/components/Activities.tsx:21]: > i think there might be a bug somewhere in the webpack code
> 
> https://deploy-preview-25--youthful-engelbart-1ad9d2.netlify.app/activities/mind-reading doesnt work. (idk if this helps but i saw this: [vuejs-templates/pwa#165](https://github.com/vuejs-templates/pwa/issues/165)) and also, in the other pr i had up, i had to add in ` publicPath: '/',` to the prod webpack

found this solution [here](https://github.com/webpack/webpack/issues/2882#issuecomment-280906981) as well.. seems like an html5 routing problem

**Comment by BryanPan342** [comment]
@reginawang99 i think we should address the html routing problems in another PR?

**Review by reginawang3495 — APPROVED** [review]
oki that sounds good to me!

**Pull request merged**